Info.plist

```diff
@@ -32,9 +34,9 @@
 				<string>srt</string>
 			</array>
 			<key>CFBundleTypeName</key>
-			<string>SubRip Subtitle</string>
+			<string>SubRip Subtitle Text Format</string>
 			<key>CFBundleTypeRole</key>
-			<string>Viewer</string>
+			<string>Editor</string>
```

Comment:
That is NOT correct. I set `Viewer`deliberately. Transcriptions can only import and export SRT files. It’s can’t edit them. This can/will lead to incorrect behavior.